### PR TITLE
[Support] Update gathering-information-for-troubleshooting-sites.md

### PR DESCRIPTION
**Pull request by ngayerie**

### CODE REVIEW

**Review by Erisa — APPROVED** [review]

File: content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md [hunk 1]
```diff
@@ -175,29 +175,27 @@ cURL is a command line tool for sending HTTP/HTTPS requests and is useful for tr
...
 
-Run the following command to send a standard HTTP GET request to your website (replace _www.example.com_ with your domain and hostname):
+Run the following command to send a standard HTTP GET request to your website (replace `www.example.com` with your hostname):
 
 ```
 $ curl -svo /dev/null http://www.example.com/
 ```
 
-This example cURL command returns output detailing the HTTP response and request headers but discards the page body output. cURL output confirms the HTTP response and whether Cloudflare is currently proxying traffic for the site.
+This example curl command returns output detailing the HTTP response and request headers but discards the page body output. curl output confirms the HTTP response and whether Cloudflare is currently proxying traffic for the site.
```
Comment (rprice74) [inline content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md:212]: Should have a capital letter at the start of a new sentence:

 Curl output confirms the HTTP response and whether Cloudflare is currently proxying traffic for the site.

Comment (Erisa) [reply content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md:212]: curl is the name of a software project thats name is canonically lowercase, so you would keep it lowercase even at the start of a new sentence. You can see on e.g. Wikipedia they do the same for curl.

File: content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md [hunk 2]
```diff
@@ -212,7 +210,7 @@ assistance.
 
 ### Performance
 
-cURL measures latency or performance degradation for HTTP/HTTPS requests via the [_\-w_ or _\--write-out_ cURL options](https://curl.haxx.se/docs/manpage.html#-w). The example cURL below measures several performance vectors in the request transaction such as duration of the TLS handshake, DNS lookup, redirects, transfers, etc:
+curl measures latency or performance degradation for HTTP/HTTPS requests via the [_\-w_ or _\--write-out_ curl option]([https://curl.haxx.se/docs/manpage.html#-w](https://curl.se/docs/manpage.html#-w)). The example curl below measures several performance vectors in the request transaction such as duration of the TLS handshake, DNS lookup, redirects, transfers, etc:
```
Comment (rprice74) [inline content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md]: If this is the start of the sentence, then we should capitalise the c in curl:

curl measures latency or performance degradation for HTTP/HTTPS requests via the [_\-w_ or _\--write-out_ curl option]([https://curl.haxx.se/docs/manpage.html#-w](https://curl.se/docs/manpage.html#-w)). The example curl below measures several performance vectors in the request transaction such as duration of the TLS handshake, DNS lookup, redirects, transfers, etc:

**Review by rprice74 — APPROVED** [review]

File: content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md [hunk 3]
```diff
@@ -212,7 +213,7 @@ assistance.
...
```
Comment (Erisa) [inline content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md]:
```suggestion
curl measures latency or performance degradation for HTTP/HTTPS requests via the [_\-w_ or _\--write-out_ curl option](https://curl.haxx.se/docs/manpage.html#-w). The example curl below measures several performance vectors in the request transaction such as duration of the TLS handshake, DNS lookup, redirects, transfers, etc:
```

File: content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md [hunk 4]
```diff
@@ -175,29 +178,27 @@ cURL is a command line tool for sending HTTP/HTTPS requests and is useful for tr
...
 ```
 
-This example cURL command returns output detailing the HTTP response and request headers but discards the page body output. cURL output confirms the HTTP response and whether Cloudflare is currently proxying traffic for the site.
+This example curl command returns output detailing the HTTP response and request headers but discards the page body output. curl output confirms the HTTP response and whether Cloudflare is currently proxying traffic for the site.
 
 {{<Aside type="note">}}
-Review the [cURL command
-options](https://curl.haxx.se/docs/manpage.html) for additional
-functionality.
+Review the [curl command options]([https://curl.haxx.se/docs/manpage.html](https://curl.se/docs/manpage.html)) for additional functionality.
```
Comment (Erisa) [inline content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md]:
```suggestion
Review the [curl command options](https://curl.se/docs/manpage.html) for additional functionality.
```

**Review by Erisa — CHANGES_REQUESTED** [review]
Some broken links were failing the checks, proposed fixes

**Review by Erisa — APPROVED** [review]

File: content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md [hunk 5]
```diff
@@ -153,19 +153,39 @@ In certain situations when request is not issued or cancelled by the browser (fo
...
+
+## Capture a NetLog dump
+
+In some cases, in order to further troubleshoot issues related to protols (errors such as `ERR_QUIC_PROTOCOL_ERROR`, `ERR_HTTP2_PROTOCOL_ERROR`, etc..) our Support team may ask you to provide a [NetLog dump](https://www.chromium.org/for-testers/providing-network-details/).
+
+{{<Aside type="warning">}}
+You can only generate a NetLog dump on the Google Chrome or Microsoft Edge browsers.
+{{</Aside>}}
+
+1.  Open a new tab and go to [about://net-export](about://net-export)
```
Comment (Erisa) [inline content/support/Troubleshooting/general-troubleshooting/gathering-information-for-troubleshooting-sites.md]: 1. Chrome doesn't let you hyperlink to an internal url, it sends you to a "blocked" page instead.
2. `about://net-export` is a redirect to `chrome://net-export` or `edge://net-export` so might as well go there directly like the below 

```suggestion
1.  Open a new tab and go to `chrome://net-export` or `edge://net-export`
```

**Pull request merged**